CHANGES.rst

```diff
@@ -23,6 +23,11 @@ associations
 
 - Changed deprecated logger.warn to logger.warning. [#2519]
 
+- Made NIRISS WFSS Level2 associations exclusive. [#2555]
```

Comment:
What exactly does "exclusive" mean in this context? You have to be a member in order to get into a NIS_WFSS association, or what?

Reply:
The original bug was that NIRISS WFSS were getting multiple associations from other rules. This is fixing that an making this rule the exclusive rule for WFSS.